src/server.c

```diff
@@ -4575,6 +4576,17 @@ int finishShutdown(void) {
                   num_replicas);
     }
 
+    if (safe && server.cluster_enabled && clusterNodeIsVotingPrimary(getMyClusterNode())) {
+        if (force) {
+            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to down. Exit anyway.");
```

Comment:
```suggestion
            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to go down. Exit anyway.");
```

Reply:
Agree, "I am" is slightly funny style. "This is" can be better.

I searched the source code and found two error replies with "I'm":

* "I'm already the owner of hash slot %u" (where %u is a number)
* "I'm a replica but my master is unknown to me"

Btw, shouldn't the first comma be a period? Command seems like bad grammar to me.

```suggestion
            serverLog(LL_WARNING, "I am a voting primary. Shutting down may cause the cluster to down. Exit anyway.");
```